spring-kafka/src/test/java/org/springframework/kafka/listener/ManualAssignmentInitialSeekTests.java

```diff
@@ -111,6 +125,12 @@ public void discardRemainingRecordsFromPollAndSeek() throws Exception {
 		public void foo(String in) {
 		}
 
+		@KafkaListener(id = "pp", autoStartup = "false",
+				topicPartitions = @org.springframework.kafka.annotation.TopicPartition(topic = "foo",
+						partitions = "#{#parsePartitions('0-5, 7, 10-15')}"))
```

Comment:
Why can't we split and parse ranges in the framework?
As long as we got a value as a string we can do with it whatever we need.

I think even that `#{'0,1,2'.split(',')}` is redundant for end-user experience...